test-requirements.txt

```diff
@@ -1,4 +1,5 @@
-ansible==2.9.19
-molecule==3.0.8
-docker==4.4.4
-ansible-lint==5.0.6
\ No newline at end of file
+ansible==5.2.0
+molecule==3.4.0
```

Comment:
Some of these packages can be upgraded, at least molecule, docker. With that upgrades, you will also need molecule-docker package, and maybe some changes at molecule.yml file.